informer/integration/test/deletionerror/deletion_error_test.go

```diff
@@ -0,0 +1,139 @@
+// +build k8srequired
+
+package deletionerror
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+)
+
+func Test_Informer_Integration_DeletionError(t *testing.T) {
```

Comment:
This is the failing test I want to make green in another PR. There is no godoc for the test but a lot of comments within the test code. I hope people can follow what this is about.